caikit/runtime/train.py

```diff
@@ -297,6 +299,9 @@ def main() -> int:
             future = train(module, wait=True, **train_kwargs)
             info = future.get_info()
             if info.status == TrainingStatus.COMPLETED:
+                complete_path = os.path.join(args.save_path, ".complete")
```

Comment:
It looks like the semantics here are only to have this file when the job completes successfully, right? That should be made clear since "complete" could also be interpreted as "done one way or another"

Reply:
That's correct. Made clear as in a code comment or did you have something else in mind?

Reply:
I think a code comment would be needed for sure. I might suggest that we call the file `.success` instead of `.complete` since `.complete` is ambiguous.

Reply:
What do others think on `.complete` vs `.success`? I have a bit of a preference for `.success` for the clarity of the word.

Reply:
I originally put `.complete` because it is an indicator of when files are done being written to the filesystem, not necessarily an indicator of job success (that would be the container exit code). We also have fms-hf-trainer using .complete, so we'll want to keep them consist it (so either we leave it as `.complete` here, or we change it to `.success` in both places.

Reply:
Ok, I guess that then brings up my other question about this only being written on successful completion then. If this is meant to indicate "files are done being written," then I think we should do it on failure as well since there may still be some files (minimally, the termination log, but possibly also intermediate checkpoints and/or log files).

Reply:
hmmmm good point.